esp/src/eclwatch/ResultsWidget.js

```diff
@@ -180,6 +193,19 @@ define([
                         }
                     }
                 });
+            } else if (params && params.legacyMode) {
+                return new DelayLoadWidget({
+                    id: id,
+                    title: "[L] " + row.Name,
```

Comment:
We use it in several places for the "L"egacy tabs (plus they will very quickly associate it with the button they just pressed)